package.json

```diff
@@ -107,7 +109,7 @@
     "check:granular-bench": "npm run build:bench && node ./test/bench/etc/run_granular_benchmarks.js",
     "check:spec-bench": "npm run build:bench && node ./test/bench/lib/spec/bsonBench.js",
     "build:bench": "cd test/bench && npx tsc",
-    "build:ts": "node ./node_modules/typescript/bin/tsc",
+    "build:ts": "node ./node_modules/typescript/bin/tsc --build --force",
```

Comment:
Adding `--build` because the project now relies on TS project references.
Adding `--force` because the `composite: true` entails `incremental: true` and subsequent runs of `build:dts` would fail because the `clean_definition_files` deletes files produced by `tsc` and because of the `incremental: true` it TSC doesn't check the existence of cached files.